openwisp_notifications/tests/test_notifications.py

```diff
@@ -576,7 +578,7 @@ def test_notification_invalid_message_attribute(self, mocked_task):
         mocked_task.assert_called_with(notification_id=notification.id)
         unregister_notification_type('test_type')
 
-    @patch.object(app_settings, 'OPENWISP_NOTIFICATIONS_HTML_EMAIL', False)
+    @patch.object(utils_settings, 'OPENWISP_HTML_EMAIL', False)
```

Comment:
These tests are already covered in openwisp-utils. I think we should remove them too :thinking:

Reply:
definitely remove tests that have been moved to openwisp-utils.